src/app.ts

```diff
@@ -11,8 +21,32 @@ winston.configure({
     ],
 });
 
-import { Matchmaker } from "./matchmaker";
+app.get("/start", bodyParser.json(), (req, res) => {
+    try {
+        winston.info(req.body);
+        mm.start();
+        res.end();
+    } catch (e) {
+        winston.error(e);
+        throw new InternalServerError(e);
```

Comment:
Errors should be passed to a common error handling service so we can put the error into a common format (one described by http-errors)

Something similar to:
```
{
statusCode: XXX,
status: "Not Found"
message: "Unable to find blah"
}
```